README.md

```diff
@@ -1 +1,9 @@
 # WebAssembly for Proxies (C++ host implementation)
+
+## Development
+
+You could use the following commands to generate the `compile_commands.json` file:
+
+```
+BAZEL_BUILD_OPTION_LIST="--define=engine=multi" ./tools/gen_compdb.py --include_all //test/... //:lib
+```
```

Comment:
Nitpicking, but this should be added to `docs/DEVELOPMENT.md` or similar and only linked to from the `README.md`. Otherwise, those files grow to contain too much noise.

Reply:
SGTM. Added a independent file and link it in the CONTRIBUTING.md